src/react/account/details/properties/__tests__/AccountInfo.test.tsx

```diff
@@ -48,17 +49,17 @@ describe('AccountInfo', () => {
       key: 'Creation Date',
       value: formatDate(new Date(account1.createDate)),
     });
-    const fifthRow = rows.at(3);
-    testRow(fifthRow, {
-      key: 'Root User Email',
-      value: account1.email,
-      extraCellComponent: 'Clipboard',
-    });
-    const sixthRow = rows.at(4);
-    testRow(sixthRow, {
-      key: 'Root User ARN',
-      value: account1.arn,
-      extraCellComponent: 'Clipboard',
-    });
+    // const fifthRow = rows.at(3);
+    // testRow(fifthRow, {
+    //   key: 'Root User Email',
+    //   value: account1.email,
+    //   extraCellComponent: 'Clipboard',
+    // });
+    // const sixthRow = rows.at(4);
+    // testRow(sixthRow, {
+    //   key: 'Root User ARN',
+    //   value: account1.arn,
+    //   extraCellComponent: 'Clipboard',
+    // });
```

Comment:
nit: we likely would want to mutualize it in the same commit as the one that comment out the Root user email and arn but I won't bother too much about that ;)